source/features/merged-pr-first-published-tag.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'dom-chef';
+import cache from 'webext-storage-cache';
+import select from 'select-dom';
+import TagIcon from 'octicon/tag.svg';
+import * as pageDetect from 'github-url-detection';
+
+import features from '../libs/features';
+import fetchDom from '../libs/fetch-dom';
+import {getRepoURL} from '../libs/utils';
+import observeElement from '../libs/simplified-element-observer';
+
+const getFirstTag = cache.function(async (commit: string): Promise<string | undefined> => {
+	const firstTag = await fetchDom<HTMLAnchorElement>(`/${getRepoURL()}/branch_commits/${commit}`, 'ul.branches-tag-list li:last-child a');
+
+	return firstTag?.textContent as string;
+}, {
+	cacheKey: ([commit]) => `first-tag:${getRepoURL()}:${commit}`
+});
+
+async function init(): Promise<void | false> {
+	const mergeCommit = select(`.TimelineItem.js-details-container.Details a[href^="/${getRepoURL()}/commit/" i] > code.link-gray-dark`)!.textContent!;
+	const tagName = await getFirstTag(mergeCommit);
+
+	if (!tagName) {
+		return false;
```

Comment:
`init` is no longer called by `features.add`, so `false` is lost. It can be dropped